.gitsubprojects

```diff
@@ -1,4 +1,4 @@
 # -*- mode: cmake -*-
-git_subproject(zeq https://github.com/HBPVis/zeq.git 812d124)
-git_subproject(FlatBuffers https://github.com/Bluebrain/flatbuffers.git ae9b391)
-git_subproject(cppnetlib https://github.com/cpp-netlib/cpp-netlib.git 0.11-devel)
```

Comment:
Nope, should stay at 0.11-devel. In fact update should not change this, I fixed this at least three million times already!